esprima.js

```diff
@@ -11,6 +11,11 @@
 var esprima = require('esprima-fb');
 var estraverse = require('estraverse-fb');
 var escodegen = require('escodegen');
+var listver = require('listver');
+
+var webpackMajorVersion = parseInt(
```

Comment:
No it's guaranteed to match webpack's loader API version but this should correspond between also webpack version 1 and 2